NDB.Covid19/NDB.Covid19/ExposureNotifications/Helpers/ExposureDetected/ExposureDetectedHelper.cs

```diff
@@ -66,5 +68,18 @@ public static void SaveLastSummary(ExposureDetectionSummary summary)
                 LogUtils.LogException(Enums.LogSeverity.ERROR, e, $"{_logPrefix}.{nameof(SaveLastSummary)}");
             }
         }
+
+        public static void SaveLastDailySummaries(IEnumerable<DailySummary>? summaries)
```

Comment:
I think we won't necessarily need it actually for anything else rather then output in DevTools, so maybe there is not much need to save it in secure storage at all and rather carry it out the same way we do it for ExposureWindows

Reply:
I agree, we can also move this method directly to dev tools